pkg/gatherers/clusterconfig/openshift_logging.go

```diff
@@ -0,0 +1,49 @@
+package clusterconfig
+
+import (
+	"context"
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/klog/v2"
+
+	"github.com/openshift/insights-operator/pkg/record"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/dynamic"
+)
+
+// GatherOpenshiftLogging collects `logging.openshift.io` resources
+// from Openshift Logging Stack.
+//
```

Comment:
I think the API is defined here https://github.com/openshift/cluster-logging-operator/blob/21ba046939b664c2e7e2640dc20f5d063f4fffc2/pkg/apis/logging/v1/clusterlogging_types.go. We can mention it in the doc